.lando/adminer.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/sh
 set -exu
 
-# Copy custom adminer.php from .lando folder to webroot.
-cp -f /app/.lando/adminer.php /var/www/html/adminer.php
+# Link custom adminer.php from .lando folder to webroot.
+ln -sf /app/.lando/adminer.php /var/www/html/
```

Comment:
@tormi I think it's better to link so for every change would wouldn't need to restart/rebuild Lando:

![image](https://github.com/wunderio/drupal-project/assets/492375/598da20c-45a9-4d8f-b98d-57053e7ff8b2)